internal/action/install.go

```diff
@@ -54,7 +55,7 @@ func Install(ctx context.Context, config *helmaction.Configuration, obj *helmv2.
 	return install.RunWithContext(ctx, chrt, vals.AsMap())
 }
 
-func newInstall(config *helmaction.Configuration, obj *helmv2.HelmRelease) (*helmaction.Install, error) {
+func newInstall(config *helmaction.Configuration, obj *helmv2.HelmRelease, opts []InstallOption) *helmaction.Install {
```

Comment:
we could have the function like:

```go
func newInstall(config *helmaction.Configuration, obj *helmv2.HelmRelease, opts ...InstallOption) *helmaction.Install {...}
```

Like this, you don't have to pass a `nil` parameter when you don't have any option.